lib/worker.js

```diff
@@ -570,28 +570,12 @@ class AssetComputeWorker {
 
     getResult(err) {
         // make sure to not return urls, customer data or credentials
+        // reduce to requestId only
 
         const result = {
-            requestId: this.params.requestId,
-            metrics: this.activationMetrics
+            requestId: this.params.requestId
```

Comment:
Removing metrics _may_ have impacts on some unit/integration tests, as I'm not sure we don't use `getResult` to check those too. But tests will tell!